modules/creating-a-machine-pool-cli.adoc

```diff
@@ -31,6 +31,7 @@ $ rosa create machinepool --cluster=<cluster-name> \
 ifdef::openshift-rosa[]
                           --disk-size=<disk_size> <8>
                           --availability-zone=<az> <9>
+                          --additional-security-group-ids <sec_group_id> <10>
```

Comment:
@bchandra-ocp This and the next change down are the other part of the machine pool creation side of XCMSTRAT-41